database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
+    minutes_item_id: str
+    name: str
+    uri: str
+    title: str
```

Comment:
Can we add an optional description field?

Reply:
Done